src/test/java/teammates/test/cases/storage/InstructorsDbTest.java

```diff
@@ -252,6 +252,37 @@ public void testGetInstructorsForCourse() {
         assertEquals(Const.StatusCodes.DBLEVEL_NULL_INPUT, ae.getMessage());
     }
 
+    @Test
+    public void testGetInstructorsDisplayedToStudents() {
+
+        ______TS("Success: get instructors displayed of a specific course to the students");
+
+        String courseId = "idOfTypicalCourse1";
+
+        List<InstructorAttributes> retrieved = instructorsDb.getInstructorsDisplayedToStudents(courseId);
+        assertEquals(4, retrieved.size());
+
+        List<String> idListOfInstructorsDisplayed = new ArrayList<>();
+        idListOfInstructorsDisplayed.add("idOfInstructor1OfCourse1");
+        idListOfInstructorsDisplayed.add("idOfInstructor2OfCourse1");
+        idListOfInstructorsDisplayed.add("idOfInstructor3");
+        idListOfInstructorsDisplayed.add(null);
+        for (InstructorAttributes instructor : retrieved) {
+            if (!idListOfInstructorsDisplayed.contains(instructor.googleId)) {
+                fail("");
```

Comment:
What are you trying to do here?

Reply:
Basically, `idListOfInstructorsDisplayed` does not contain any of the instructor retrieved from calling`getInstructorsDisplayedToStudents(courseId)` in the instructorDb , an assertion is thrown. Similar to the previous test case `testGetInstructorsForCourse()`